R/cache.R

```diff
@@ -50,17 +50,7 @@ font_cache_housing <- function() {
   )
 }
 
-# Intentionally mimics `tools::R_user_dir("thematic", "cache")` (coming in R 4.0)
-# Copyright (C) 2020 The R Core Team
-# https://github.com/wch/r-source/blob/trunk/src/library/tools/R/userdir.R
+
 thematic_cache_dir <- function() {
-  home <- normalizePath("~")
-  path <- if (.Platform$OS.type == "windows") {
-    file.path(Sys.getenv("LOCALAPPDATA"), "R", "cache")
-  } else if (Sys.info()["sysname"] == "Darwin") {
-    file.path(home, "Library", "Caches", "org.R-project.R")
-  } else {
-    file.path(home, ".cache")
-  }
-  file.path(path, "R", "thematic")
+  rappdirs::user_cache_dir("thematic")
```

Comment:
Maybe this should work more like `sass::sass_cache_context_dir()`. It would remove the need to copy over the (entire) cache in the pre deploy hook, which would lead to less space wastage